workbench-core/authorization/src/dynamicAuthorization/dynamicAuthorizationService.test.ts

```diff
@@ -454,6 +458,20 @@ describe('DynamicAuthorizationService', () => {
     });
   });
 
+  test('get identity permissions by identity', async () => {
+    mockDynamicAuthorizationPermissionsPlugin.getIdentityPermissionsByIdentity = jest.fn().mockResolvedValue({
+      data: {
+        identityPermissions: [mockIdentityPermission]
+      }
+    });
+    const request = {
+      identityId: sampleGroupId,
+      identityType: sampleGroupType
+    };
+
+    const { data } = await dynamicAuthzService.getIdentityPermissionsByIdentity(request);
+    expect(data.identityPermissions).toStrictEqual([mockIdentityPermission]);
+  });
```

Comment:
I think this test got separated from its parent `describe` in the merge from develop